v63005/outref/gtm8980.txt

```diff
@@ -0,0 +1,66 @@
+# Running v63005/gtm8980.csh to test that rare uses of VIEW and $VIEW are handled correctly 
+
+# Create a 3 region DB with gbl_dir mumps.gld and regions DEFAULT, AREG, and BREG
+
+
+# GTM-8980 calls for testing calls of VIEW and $VIEW with an empty string
```

Comment:
Add the following release note line here. "VIEW and $VIEW() with a empty string or inappropriate region-list works appropriately; in V6.3-004 these could cause inappropriate results, including a segmentation violation (SIG-11). "